apps/bandwagon/templates/bandwagon/includes/addedit.html

```diff
@@ -1,6 +1,11 @@
 {{ csrf() }}
 
 <fieldset>
+  {# Honeypot field for bots to get caught #}
+  {% if form.your_name %}
```

Comment:
Isn't the `CollectionForm` also used for editing? I'm not arguing over the `if`, just curious.